File: src/main/java/webview/WebviewThread.java
package webview;

import com.sun.jna.Pointer;

public class WebviewThread extends Thread {

    private final Pointer windowPointer;
    private final WebviewLibrary webViewLib;

    public WebviewThread(WebviewLibrary webViewLib, Pointer windowPointer) {
        this.webViewLib = webViewLib;
        this.windowPointer = windowPointer;
    }

    @Override
    public void run() {
        System.out.println("7");
        webViewLib.webview_run(windowPointer);
        System.out.println("8");
        boolean run = true;
        while (run) {
            try {
                Thread.sleep(1000);
            } catch (InterruptedException e) {
                e.printStackTrace();
                run = false;
            }
        }
        webViewLib.webview_destroy(windowPointer);
    }
}

File: src/main/java/webview/WebviewThread.kt
package webview

import com.sun.jna.Pointer

class WebviewThread(private val webViewLib: WebviewLibrary, private val windowPointer: Pointer) : Thread() {
    override fun run() {
        println("7")
        webViewLib.webview_run(windowPointer)
        println("8")
        var run = true
        while (run) {
            try {
                sleep(1000)
            } catch (e: InterruptedException) {
                e.printStackTrace()
                run = false
            }
        }
        webViewLib.webview_destroy(windowPointer)
    }
}